downloads/_downloads.php

```diff
@@ -16,9 +16,7 @@
 
 <li>Ubuntu, Wasta-Linux: Keyman for Linux can be installed via launchpad:</li>
 <blockquote><pre class='language-bash code'><code>sudo add-apt-repository ppa:keymanapp/keyman
-sudo apt-get update
-sudo apt-get upgrade
-sudo apt-get install keyman ibus-keyman onboard</code></pre></blockquote>
+sudo apt install keyman onboard-keyman</code></pre></blockquote>
```

Comment:
Since `onboard-keyman` won't be available on ppa:keyman, would it be better leave `onboard` here until we release to stable? (hmm, or I could upload `onboard-keyman` to ppa:keyman - which wouldn't mess up anything since it's a new package).

Reply:
No, this is fine; it's on the staging site which we will deploy to production at time of release.